doc/source/oss-users.rst

```diff
@@ -0,0 +1,80 @@
+.. _ray-oss-list:
+
+Open source projects that run on Ray
+====================================
+
+The Ray ecosystem is rapidly growing! This page lists projects that are known to use Ray. This does not constitute an endorsement.
+
+*Using Ray?* `Fill out this form <https://forms.gle/ps4S1C955kjotxd57>`__ *and we'll be in contact to get added to the list.*
```

Comment:
```suggestion
*Using Ray?* `Fill out this form <https://forms.gle/ps4S1C955kjotxd57>`__ *and we'll be in contact to get added to the list.*
```

Reply:
there doesn't seem to be a change here suggested so i'll just resolve